src/loaders.js

```diff
@@ -34,6 +42,8 @@ export const loadMtl = mtlPath => new Promise((resolve, reject) => {
   const loader = new global.THREE.MTLLoader();
 
   loader.setTexturePath(mtlPath.substring(0, mtlPath.lastIndexOf('/') + 1));
+  // remaps ka, kd, & ks values of 0,0,0 -> 1,1,1
+  loader.setMaterialOptions( { ignoreZeroRGBs: true } );
```

Comment:
style nit: no spaces outside of the curly `setMaterialOptions({ ignoreZeroRGBs: true })`

Reply:
done